source/documentation/common/install/proc-Configuring_the_Red_Hat_Virtualization_Manager.adoc

```diff
@@ -56,13 +56,6 @@ Set up Cinderlib integration
 (Currently in tech preview)
```

Comment:
This line has probably been removed from the code. It should be checked.

Reply:
right! 
need to verify with Dev...